CHANGELOG.md

```diff
@@ -6,11 +6,21 @@
 
 ### Changed
 
+### Fixed
+
+
+<a id="1.34.0"></a>
```

Comment:
```suggestion
```

Reply:
I'm pretty sure it works without.

Reply:
Oh but the default anchor is annoyingly long.

Reply:
I think the "right" solution would be having a short-link on a deltachat domain

Reply:
```suggestion
<a id="1_34_0"></a>
```

Reply:
actually I think github is just weird, dots in anchors work as good as underscores, but I changed to underscores now anyway